solara/website/pages/documentation/examples/utilities/countdown_timer.py

```diff
@@ -41,24 +41,26 @@ def run_timer():
 
     solara.use_thread(run_timer, dependencies=[duration, running])
 
-    with solara.VBox() as main:
-        if not running:
-            if duration < 1:
-                solara.Error("Duration must be at least 1 second")
-            else:
-                solara.Markdown(f"# Timer set to {seconds} seconds")
+    # with solara.VBox() as main:
+    solara.Column()
+    if not running:
+        if duration < 1:
+            solara.Error("Duration must be at least 1 second")
         else:
-            if seconds:
-                solara.Markdown(f"# {seconds} seconds left")
-            else:
-                solara.solara.Markdown("# Time's up!")
-        rv.TextField(type="number", v_model=duration, on_v_model=on_duration, disabled=running)
-        with solara.HBox():
-            if running:
-                solara.Button("Stop", on_click=lambda: set_running(False), icon_name="mdi-stop")
+            solara.Markdown(f"# Timer set to {seconds} seconds")
+    else:
+        if seconds:
+            solara.Markdown(f"# {seconds} seconds left")
+        else:
+            solara.solara.Markdown("# Time's up!")
+    # rv.TextField(type="number", v_model=duration, on_v_model=on_duration, disabled=running)
+    solara.v.TextField(type="number", v_model=duration, on_v_model=on_duration, disabled=running)
```

Comment:
The top one can be removed.
It might be that we can use solara.InputInt or solara.InputFloat. When we wrote this example, those didn't exist yet.